tests/unit/source_code/test_notebook_linter.py

```diff
@@ -585,3 +589,51 @@ def test_computes_values_across_notebooks_using_run(extended_test_index, mock_pa
         )
     ]
     assert advices == expected
+
+
+def test_ignores_values_across_notebooks_using_dbutils_notebook_run(extended_test_index, mock_path_lookup):
+    path = mock_path_lookup.resolve(Path("values_across_notebooks_dbutils_notebook_run.py"))
+    source = path.read_text()
+    linter = NotebookLinter.from_source(
+        extended_test_index, mock_path_lookup, CurrentSessionState(), source, Language.PYTHON
+    )
+    advices = list(linter.lint())
+    expected = [
+        Advisory(
+            code='table-migrate-cannot-compute-value',
+            message="Can't migrate 'spark.table(f'{a}')' because its table name argument cannot be computed",
+            start_line=3,
+            start_col=0,
+            end_line=3,
+            end_col=19,
+        ),
+        Advice(
+            code='table-migrate',
+            message='The default format changed in Databricks Runtime 8.0, from Parquet to Delta',
+            start_line=3,
+            start_col=0,
+            end_line=3,
+            end_col=19,
+        ),
+    ]
+    assert advices == expected
+
+
+def test_computes_values_across_notebooks_using_magic_line(extended_test_index, mock_path_lookup):
```

Comment:
can you rewrite this as a functional test? they're easier to read. and maintain. and write as well.

Reply:
done